docs/design_proposals/ko-builder.md

```diff
@@ -259,14 +263,20 @@ Adding the ko builder requires making config changes to the Skaffold schema.
     	// Dependencies are the file dependencies that skaffold should watch for both rebuilding and file syncing for this artifact.
     	Dependencies *KoDependencies `yaml:"dependencies,omitempty"`
 
+      // Dir is the directory where the `go` tool will be run.
+      // The value is a directory path relative to the `context` directory.
+      // If empty, the `go` tool will run in the `context` directory.
+      // Example: `my-go-mod-is-in-this-dir`
```

Comment:
I hoped to come up with a more realistic example, but I can't think of anything that is any better (`live-at-head`, `compat-go114`)

Reply:
Those are better examples, I've updated the doc.